security/advancedtls/crl_provider.go

```diff
@@ -136,7 +136,7 @@ func (o *FileWatcherOptions) validate() error {
 	}
 	// Checks related to RefreshDuration.
 	if o.RefreshDuration <= 0 {
-		grpclogLogger.Warningf("RefreshDuration is not set or negative: provided value %v, default value %v will be used.", o.RefreshDuration, defaultCRLRefreshDuration)
+		grpclogLogger.Infof("RefreshDuration is not set or negative: provided value %v, default value %v will be used.", o.RefreshDuration, defaultCRLRefreshDuration)
```

Comment:
Actually I don't think this necessarily needs to be logged at all?  Especially in the case where you're just using the default, anything being logged would be spam.  Maybe if you had a negative number?  But who would even do such a thing?  IMO:

```go
if dur == 0 {
	// use default, silently
} else {
	// use 1m and info or warning that we're using 1m instead of the illegal value they used
}
```

See https://github.com/grpc/grpc-go/blob/master/Documentation/log_levels.md for some guidance we try to follow regarding this.

Reply:
Done